ProjectLighthouse/Configuration/ServerConfiguration.cs

```diff
@@ -185,6 +185,7 @@ private void writeConfig(string path)
     public string AnnounceText { get; set; } = "You are now logged in as %user (id: %id).";
 #endif
     public bool CheckForUnsafeFiles { get; set; } = true;
+    public bool LogChatFiltering { get; set; } = false;
```

Comment:
This is fine but when you modify or add any variables in the config you need to bump the `CurrentConfigVersion` field so that people's configurations will be properly migrated